mgmt/rest/server.go

```diff
@@ -80,6 +80,38 @@ type Config struct {
 	RestAuthPassword string `json:"rest_auth_password,omitempty"yaml:"rest_auth_password,omitempty"`
 }
 
+const (
+	CONFIG_CONSTRAINTS = `
+	"restapi" : {
+					"type": ["object", "null"],
+					"properties" : {
+									"enable": {
+													"type": "boolean"
+									},
+									"https" : {
+													"type": "boolean"
+									},
+									"rest_auth": {
+													"type": "boolean"
+									},
+									"rest_auth_password": {
+													"type": "string"
+									},
+									"rest_certificate": {
+													"type": "string"
+									},
+									"rest_key" : {
+													"type": "string"
+									},
+									"port" : {
+													"type": "integer"
```

Comment:
FWIW binding to port 0 usually means any unused ephemeral port in Linux-land.

> On May 4, 2016, at 13:25, Nicholas Weaver <notifications@github.com> wrote:
> 
> In mgmt/rest/server.go:
> 
> > +													"type": "boolean"
> > +									},
> > +									"rest_auth": {
> > +													"type": "boolean"
> > +									},
> > +									"rest_auth_password": {
> > +													"type": "string"
> > +									},
> > +									"rest_certificate": {
> > +													"type": "string"
> > +									},
> > +									"rest_key" : {
> > +													"type": "string"
> > +									},
> > +									"port" : {
> > +													"type": "integer"
> Should be greater than zero at least.
> 
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
>